modules/users/client/views/profile/profile-edit-account.client.view.html

```diff
@@ -137,6 +137,44 @@
 </div>
 <!-- /Password -->
 
+<!-- Notifications -->
+<div class="panel panel-default">
+  <div class="panel-heading">
+    Notifications
+  </div>
+  <div class="panel-body">
+
+    <form autocomplete="off">
+      <div class="form-horizontal">
+
+        <div class="form-group">
+          <label class="col-sm-3 text-right control-label">Push</label>
+          <div class="col-sm-9">
+
+            <div class="checkbox">
+              <label>
+                <input type="checkbox"
+                       name="browser"
+                       ng-model="profileEditAccount.pushEnabled"
+                       ng-disabled="profileEditAccount.pushIsDisabled()"
+                       ng-change="profileEditAccount.pushUpdate()">
+                Current web browser
+              </label>
+            </div>
+
+            <p class="help-block" ng-if="profileEditAccount.push.isSupported"><small>This will send notifications even when the website is closed.</small></p>
+            <p class="help-block" ng-if="!profileEditAccount.push.isSupported"><small>Your browser does not support push notifications.</small></p>
```

Comment:
The section could go above password tho, so it would be more noticeable.

![image](https://cloud.githubusercontent.com/assets/87168/25010406/b91b9ae2-2072-11e7-9013-e8ef76741716.png)